pandas/core/window/rolling.py

```diff
@@ -234,17 +234,16 @@ def _validate_get_window_bounds_signature(window: BaseIndexer) -> None:
                 f"get_window_bounds"
             )
 
-    def _create_blocks(self, obj: FrameOrSeriesUnion):
```

Comment:
can you rename these then as well, maybe _construct_data or similar

Reply:
sure